### PR TITLE
Fix:-Documentation typo fixes

### LINKED ISSUE #327: Bug Report: Typos in the Documentation
Going through the current talawa docs i have found quite a number of typos in the documentation and I would like to fix them

### PR DESCRIPTION
**Pull request by Biki-das**
fix #327 

I have installed a third party plugin to scan for typos . so it listed out the possible one, if anyone find other typos they can fix them referring the same issue i created

### CODE REVIEW

**Review by palisadoes — APPROVED** [review]

File: docs/developers/talawa-admin/introduction.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ title: Overview
 
 #### 3. `Events` Page
 
-- This shows list of active `Events` in the organization.
+- These shows list of active `Events` in the organization.
```
Comment (palisadoes) [inline docs/developers/talawa-admin/introduction.md:22]: `These show a list`

File: docs/features/members/preapprovals.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ TBD.
 
 3. One time password/token given to the user generated by admins
 
- Note: Ideally the QR code should contain all the organizations API login information so that it doesn&#39;t need to be rekeyed.
+ Note: Ideally the QR code should contain all the organizations API login information so that it doesn&#39;t need to be re keyed.
```
Comment (palisadoes) [inline docs/features/members/preapprovals.md]: `re-keyed`

File: docs/features/members/roles.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ This feature aims to _ **introduce application roles** _ by _ **defining how the
 
 ### Problem
 
-1. **Members need to be assigned roles :** in order to provide a tailored a experience for each user so that application features are not arbitrarily available to all users of the application
+1. **Members need to be assigned roles :** in order to provide a tailored an experience for each user so that application features are not arbitrarily available to all users of the application
```
Comment (palisadoes) [inline docs/features/members/roles.md]: `a tailored experience`

**Review by palisadoes — CHANGES_REQUESTED** [review]
Please make the changes requested.

**Comment by palisadoes** [comment]
What package did you use to do this? We can ask someone to automate the processing with each pull request and get a overall higher quality of documentation. @tasneemkoushar @evasharma12 @anwersayeed

**Comment by Biki-das** [comment]
@palisadoes thanks for the quick review!

**Comment by palisadoes** [comment]
What package did you use for this? We need to get it integrated with our workflows. A manual process isn't sustainable.

**Comment by Biki-das** [comment]
> What package did you use for this? We need to get it integrated with our workflows. A manual process isn't sustainable.

https://github.com/streetsidesoftware/vscode-spell-checker , this is the one i used!

**Comment by evasharma12** [comment]
> What package did you use to do this? We can ask someone to automate the processing with each pull request and get a overall higher quality of documentation. @tasneemkoushar @evasharma12 @anwersayeed

@palisadoes We didn't use any package for spell check or any typo mistakes. But using a package to automate this process with each PR might create problem. There are some technical or code related words which might not pass with the spell check. So I think we can use some package to only highlight the errors and let the writer/coder handle them manually.

**Comment by Biki-das** [comment]
> > What package did you use to do this? We can ask someone to automate the processing with each pull request and get a overall higher quality of documentation. @tasneemkoushar @evasharma12 @anwersayeed
> 
> @palisadoes We didn't use any package for spell check or any typo mistakes. But using a package to automate this process with each PR might create problem. There are some technical or code related words which might not pass with the spell check. So I think we can use some package to only highlight the errors and let the writer/coder handle them manually.

the spell checker sometime catches word which are not typos, so i agree on this , adding it as a PR filter might lead to problems!

**Review by palisadoes — APPROVED** [review]

**Comment by Biki-das** [comment]
> > What package did you use to do this? We can ask someone to automate the processing with each pull request and get a overall higher quality of documentation. @tasneemkoushar @evasharma12 @anwersayeed
> 
> @palisadoes We didn't use any package for spell check or any typo mistakes. But using a package to automate this process with each PR might create problem. There are some technical or code related words which might not pass with the spell check. So I think we can use some package to only highlight the errors and let the writer/coder handle them manually.

@evasharma12 we can add those technical words as a list of valid words in a config file and then run the Gh workflow, this is a more better work flow

cc @palisadoes

**Pull request merged**